superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -357,6 +658,7 @@ export default class ResultSet extends React.PureComponent<
     if (query.progress > 0) {
       progressBar = (
         <ProgressBar
+          // @ts-ignore
```

Comment:
are you running into issues where a value is missing on these types/interfaces?

Reply:
Yea wasn't able to figure out what the issue is